Cargo.lock

```diff
@@ -1043,7 +1043,7 @@ dependencies = [
 
 [[package]]
 name = "humility"
-version = "0.9.13"
+version = "0.9.14"
```

Comment:
This will conflict with https://github.com/oxidecomputer/humility/pull/289, so we'll see who gets merged first

Reply:
#289 landed first, so I updated to 0.9.15